src/Page.php

```diff
@@ -811,6 +812,33 @@ public function assertNotClosed(): void
         }
     }
 
+    /**
+     * Set user agent for the current page.
+     * @see https://source.chromium.org/chromium/chromium/deps/icu.git/+/faee8bc70570192d82d2978a71e2a615788597d1:source/data/misc/metaZones.txt | ICU’s metaZones.txt
+     *
+     * @throws InvalidTimezoneId|CommunicationException|NoResponseAvailable
+     *
+     */
+    public function setTimezone($timezoneId = null)
+    {
+        // ensure target is not closed
+        $this->assertNotClosed();
+
+        $response = $this->getSession()
+            ->sendMessageSync(
+                new Message(
+                    'Emulation.setTimezoneOverride',
+                    [
+                        'timezoneId' => $timezoneId ?? '',
+                    ]
+                )
+            );
+
+        if (mb_strpos($response->getErrorMessage(), 'Invalid timezone')) {
```

Comment:
why mb_strpos?

Reply:
Because the error message has some more characters as prefix

Reply:
i mean as opposed to strpos

Reply:
we don't depend on the mbstring extension

Reply:
Oh I think there is no difference so I can update to use `strpos`